File: HelloWorld.java
import java.io.*
import java.util.*

public class Solution {
	public static void main(String[] args) {
        // Create a Scanner object to read input from stdin.
		Scanner scan = new Scanner(System.in); 
		
		// Read a full line of input from stdin and save it to our 
variable, inputString.
		String inputString = scan.nextLine(); 

		// Close the scanner object, because we've finished reading 
********* Important to avoid RESOURCE LEAK **********
        // all of the input from stdin needed for this challenge.
		scan.close(); 
      
		// Print a string literal saying "Hello, World." to stdout.
		System.out.println("Hello, World.");
      
	    System.out.print(inputString);
	}
}

File: HelloWorld.c
#include<stdio.h>
#include<string.h>
#include<stdlib.h>
int main() {
    // Declare a variable named 'input_string' to hold our input.
    char input_string[105]; 
    
    // Read a full line of input from stdin and save it to our variable, 
input_string.
    
    scanf("%[^\n]", input_string); 
    
    /* 
*******************************************************************************
     %s is not used because it will not read the whole input line & will take 
only thefirst word just before the first
     whitespace after that it would've dropped it.
     
    
**********************************************************************************  
*/
    
    // Print a string literal saying "Hello, World." to stdout using printf.
    printf("Hello, World.\n");
    
    printf("%s",input_string);
    
    return 0;
}
